pkg/authorization/registry/rolebinding/rest.go

```diff
@@ -73,6 +73,9 @@ func (r *REST) Create(ctx kapi.Context, obj runtime.Object) (runtime.Object, err
 	if err := r.validateReferentialIntegrity(ctx, roleBinding); err != nil {
 		return nil, err
 	}
+	if err := r.confirmRoleBindingNameUnique(ctx, roleBinding.Name); err != nil {
+		return nil, err
+	}
```

Comment:
I agree. I'll do it inside of the new rolebiding.VirtualRegistry either in or after prevent escalation